cmake/TorchVisionConfig.cmake.in

```diff
@@ -22,23 +22,26 @@ if(NOT (CMAKE_VERSION VERSION_LESS 3.0))
 # Don't include targets if this file is being picked up by another
 # project which has already built this as a subproject
 #-----------------------------------------------------------------------------
-if(NOT TARGET ${PN}::TorchVision)
+if(NOT TARGET ${PN}::${PN})
 include("${CMAKE_CURRENT_LIST_DIR}/${PN}Targets.cmake")
 
-if(NOT TARGET torch_library)
+target_include_directories(${PN}::${PN} INTERFACE "${${PN}_INCLUDE_DIR}" ${JPEG_INCLUDE_DIRS} ${PNG_INCLUDE_DIRS})
+
 find_package(Torch REQUIRED)
-endif()
+find_package(PNG REQUIRED)
+find_package(JPEG REQUIRED)
```

Comment:
This makes PNG/JPEG mandatory dependencies. Currently those are [optional](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#install-torchvision) dependencies.

Reply:
Honestly, I don't think this could be considered optional dependencies because in

https://github.com/pytorch/vision/blob/main/CMakeLists.txt#L27-L28

will always make these two libraries mandatory unless we provide extra options in the main CMakeLists.txt

Reply:
Well you do have a point on that.

Reply:
I will push another commit to make these consistent.